pkg/database/mariadb.go

```diff
@@ -55,6 +55,14 @@ func (info MariaDBDatabaseInfo) GetWriteResourceSQL() string {
 		"ON DUPLICATE KEY UPDATE name=?, namespace=?, resource_version=?, cluster_deleted_ts=?, data=?"
 }
 
+func (info MariaDBDatabaseInfo) GetWriteUrlSQL() string {
+	return "INSERT INTO log_url (uuid, url) VALUES (?, ?)"
```

Comment:
I am currently focused on addressing the retrieval of the log URLs without the container filter but based on a previous discussion I think that we agreed to add a `container` column on the `log_url` table and that the sink should introduce it so it's easier to make the filter.

This is a quite large PR so I'm ok with addressing it separately